controllers/tagsController.js

```diff
@@ -0,0 +1,34 @@
+import models from '../models';
+import ResponseHandler from '../helpers/responseHelper';
+import { STATUS } from '../helpers/constants';
+
+/**
+ * Class representing tags controller
+ *
+ * @class TagsController
+ */
+class TagsController {
+  /**
+   * Creates a new user resource
+   *
+   * @static
+   * @param {Request} request - Request object
+   * @param {Response} response - Response object
+   * @param {Function} next - Express next function
+   * @returns {void}
+   *
+   * @memberof TagsController
+   */
+  static async index(request, response, next) {
```

Comment:
#### Nitpick
- Thought we discussed how you name functions earlier when you created the `User` model

Reply:
That was my fault, I'll resolve this to a descriptive name. I think `getTags` will be fine.